README.md

```diff
@@ -86,6 +88,35 @@ $ npm config set script-shell "C:\\Program Files\\Git\\bin\\bash.exe"
 
 You must have [Git for Windows](https://git-scm.com/download/win) installed. See this [StackOverflow answer](https://stackoverflow.com/a/46006249) for more details.
 
+### Build the bot database
+
+
+*As we use prisma for managing our DB it is up to you what relational database framework you use.*
+
+[The following is a guide to setting up Postgres inside a docker container.](https://github.com/docker-library/docs/blob/master/postgres/README.md) While the choice of database is up to you, the instructions for getting started in this guide assume you are using a postgres docker image. 
+After you have Postgres (or your DB of choice) up and running edit the line in your .env file to be 
+```
+DATABASE_URL=postgres://pg_user:pg_pass@pg_host:pg_port/pg_table
+# required for any DB functionality, we will get this URL in a later section
+```
+- pg_user  = The Username you set in your POSTGRES_USER environment variable (default postgres)
+- pg_pass  = The Password you set in your POSTGRES_PASS environment variable (default postgres)
```

Comment:
Are these environment variables meant to go in the local `.env` file, or are they defined on the Postgres server? Should users include `POSTGRES_USER` and `POSTGRES_PASS` as separate entries in this local `.env` file, or should all of that be in `DATABASE_URL`?

Are these listed here only to explain the contents of `DATABASE_URL`? If so, would it not be easier to rewrite the example `DATABASE_URL` variable to use inline placeholders, like so:
```sh
DATABASE_URL=postgres://{USERNAME}:{PASSWORD}@{HOST}:{PORT}/pg_table
```

Also, Prisma operates on multiple tables, doesn't it? Would specifying one in the connection URL break things?

Reply:
I believe those 2 lines are just to explain the connection string, you still end up with just the db url in your.

postgresql://<username>:<password>@<hostname>:<port>/<database> is the format for the connection string, so the pg_table variable indicates the database with multiple tables, not just a single table. However, it should probably be changed to pg_db or something similar to properly indicate that.